mocha/test/version.test.js

```diff
@@ -0,0 +1,22 @@
+const httpStatus = require("http-status");
+const APP_URL = process.env.MOCHA_URL;
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+
+let apiEndPoint = apiEndPoints.versionApiEndpoint;
+
+describe("GET Version endpoint test", () => {
+  it("sending GET request with endpoint, status : 200 OK with response body", async () => {
+    let response = await apiHelper.sendGETRequest(APP_URL, apiEndPoint);
+    validator.validateResponse(response, {
+      statusCode: httpStatus.OK,
+    });
+
+    validator.validateResponse(response.body, {
+      commit: "unknown",
```

Comment:
Hi @adorapham 
Maybe we could create an issue and ask why we have commit:"unknown". What is the purpose of this? Maybe "unknown" should be replaced by a value?

Reply:
@galeza , I create [PR#2121](https://github.com/FightPandemics/FightPandemics/issues/2121) like a question back to developers.